### PR TITLE
Add to lang

### PR DESCRIPTION
**Pull request by ExaInsanity**
Add a discord invite to the message prompting you to join the support discord

### CODE REVIEW

**Comment by BitesizedLion** [comment]
pls do <> and put the link inside it

like <https://moron.com>

File: discord/src/main/resources/lang/English (UK).yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ values:
   couldnt-find-setting: ":x: Could not find that setting. Use `>preset settings` if you're unsure."
   no-active-giveaways: ":x: There are no active giveaways in this server."
   no-scheduled-giveaways: ":x: There are no scheduled giveaways in this server."
-  fatal-error-loading-server: ":warning: There was a fatal error loading your server. Please join the support discord and scream at Zak."
+  fatal-error-loading-server: ":warning: There was a fatal error loading your server. Please join the support discord at https://discord.gg/ancjpjEEPX and scream at Zak."
```
Comment (BitesizedLion) [inline discord/src/main/resources/lang/English (UK).yml]: pls do <> and put the link inside it

like https://moron.com

**Review by BitesizedLion — APPROVED** [review]
looks good

**Review by ZakShearman — APPROVED** [review]

File: discord/src/main/resources/lang/English (UK).yml (hunk 1 above)
Comment (ExaInsanity) [reply discord/src/main/resources/lang/English (UK).yml]: Fixed in 80a7cfc387e689e32f0a1c1f8100a96378ca0d03

**Pull request merged**